Automattic/jetpack
# Launchpad: Update the URL of 'Enable subscribers modal' task

![Markup on 2023-08-17 at 09:57:17](https://github.com/Automattic/jetpack/assets/2019970/60717d16-5e34-488a-bfd6-df8d2dc860b3)


## Proposed changes:

* Update the URL of the "Enable subscribers modal" task

### Other information:


## Jetpack product discussion


- p7DVsv-ioX-p2#comment-47038
- p1692248242076719-slack-C02TCEHP3HA

## Does this pull request change what data or activity we track or use?


No, it does not. 

## Testing instructions:






* Apply the PR to your sandbox.
* Ensure that you have a test site where the current theme is `lettre`, or the `site_intent` blog option is set to `newsletter` and the site has a block theme applied 
  * 'site_intent` can be modified with the aid of `wpsh` as such:
  ```
    switch_to_blog( $test_blog_id);
    update_option( 'site_intent', 'newsletter' );
  ```
* Sandbox `public-api.wordpress.com`
* Do an API test call to `/wpcom/v2/sites/$your-test-site-slug/launchpad?checklist_slug=intent-free-newsletter`
   ![Markup on 2023-08-17 at 12:20:49](https://github.com/Automattic/jetpack/assets/2019970/36b7e917-d8c0-4db4-b869-5fa7552dcb9d)
